From c344b0396b4ef4435860eb6e3954972e028e3491 Mon Sep 17 00:00:00 2001
From: "Bryn M. Reeves" <bmr@redhat.com>
Date: Fri, 31 Jan 2014 15:25:15 +0000
Subject: [PATCH 02/72] Remove obsolete diagnostics code from ldap plugin

Signed-off-by: Bryn M. Reeves <bmr@redhat.com>
---
 sos/plugins/ldap.py | 12 ------------
 1 file changed, 12 deletions(-)

diff --git a/sos/plugins/ldap.py b/sos/plugins/ldap.py
index 361de73..f1032a9 100644
--- a/sos/plugins/ldap.py
+++ b/sos/plugins/ldap.py
@@ -22,18 +22,6 @@ class Ldap(Plugin):
     plugin_name = "ldap"
     ldap_conf = "/etc/openldap/ldap.conf"
 
-    def get_ldap_opts(self):
-        # capture /etc/openldap/ldap.conf options in dict
-        # FIXME: possibly not hardcode these options in?
-        ldapopts=["URI","BASE","TLS_CACERTDIR"]
-        results={}
-        tmplist=[]
-        for i in ldapopts:
-            t=self.do_regex_find_all(r"^(%s)\s+(.*)" % i,self.ldap_conf)
-            for x in t:
-                results[x[0]]=x[1].rstrip("\n")
-        return results
-
     def setup(self):
         super(Ldap, self).setup()
         self.add_copy_spec("/etc/ldap.conf")
-- 
1.9.3

