From c73207037d86a96550b1e0aca9fb8172f4e68754 Mon Sep 17 00:00:00 2001
From: "Bryn M. Reeves" <bmr@redhat.com>
Date: Mon, 3 Feb 2014 11:48:46 +0000
Subject: [PATCH 05/72] Fix get_option() use in cluster plugin

Signed-off-by: Bryn M. Reeves <bmr@redhat.com>
---
 sos/plugins/cluster.py | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/sos/plugins/cluster.py b/sos/plugins/cluster.py
index 0fc4ded..eeacdab 100644
--- a/sos/plugins/cluster.py
+++ b/sos/plugins/cluster.py
@@ -90,11 +90,11 @@ class Cluster(Plugin, RedHatPlugin):
                     - timedelta(hours=72)).strftime("%Y-%m-%d %H:%m:%S")
         if self.get_option('crm_from') != False:
             if re.match(r'\d{4}-\d{2}-\d{2} \d{2}:\d{2}:\d{2}',
-                        str(self.getOption('crm_from'))):
-                crm_from = self.getOption('crm_from')
+                        str(self.get_option('crm_from'))):
+                crm_from = self.get_option('crm_from')
             else:
                 self.soslog.error("crm_from parameter '%s' is not a valid date"
-                            % self.getOption('crm_from'))
+                            % self.get_option('crm_from'))
 
         crm_dest = os.path.join(self.get_cmd_dir(), 'crm_report')
         self.add_cmd_output('crm_report -S -d --dest %s --from "%s"'
-- 
1.9.3

