From f617db3b8232c5864726c60b11a91162409d867f Mon Sep 17 00:00:00 2001
From: "Bryn M. Reeves" <bmr@redhat.com>
Date: Thu, 20 Mar 2014 18:56:58 +0000
Subject: [PATCH 37/72] Call rhsm-debug with the --sos switch

Signed-off-by: Bryn M. Reeves <bmr@redhat.com>
---
 sos/plugins/yum.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/sos/plugins/yum.py b/sos/plugins/yum.py
index c978842..b05c0fe 100644
--- a/sos/plugins/yum.py
+++ b/sos/plugins/yum.py
@@ -48,7 +48,7 @@ class Yum(Plugin, RedHatPlugin):
             "/var/log/rhsm/rhsmcertd.log"])
         self.add_cmd_output("subscription-manager list --installed")
         self.add_cmd_output("subscription-manager list --consumed")
-        self.add_cmd_output("rhsm-debug system --no-archive --destination %s"
+        self.add_cmd_output("rhsm-debug system --sos --no-archive --destination %s"
                 % self.get_cmd_output_path())
 
         if self.get_option("yumlist"):
-- 
1.9.3

