From 36055d3d069a1176787e4dfb722fc5ca9a804ac5 Mon Sep 17 00:00:00 2001
From: "Bryn M. Reeves" <bmr@redhat.com>
Date: Thu, 27 Mar 2014 13:46:40 +0000
Subject: [PATCH 50/72] Fix x86 arch detection in processor plugin

Signed-off-by: Bryn M. Reeves <bmr@redhat.com>
---
 sos/plugins/processor.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/sos/plugins/processor.py b/sos/plugins/processor.py
index 0b236f8..14d800e 100644
--- a/sos/plugins/processor.py
+++ b/sos/plugins/processor.py
@@ -36,7 +36,7 @@ class Processor(Plugin, RedHatPlugin, UbuntuPlugin, DebianPlugin):
         self.add_cmd_output("cpupower idle-info")
         self.add_cmd_output("cpupower frequency-info")
 
-        if self.policy().get_arch().endswith("386"):
+        if '86' in self.policy().get_arch():
             self.add_cmd_output("x86info -a")
 
 
-- 
1.9.3

