From cfefd80c828c309745cc40d8498223b4fbc7b5ca Mon Sep 17 00:00:00 2001
From: "Bryn M. Reeves" <bmr@redhat.com>
Date: Thu, 27 Mar 2014 20:52:49 +0000
Subject: [PATCH 58/72] Update Plugin tests to treat copy_paths as a set

Signed-off-by: Bryn M. Reeves <bmr@redhat.com>
---
 tests/plugin_tests.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tests/plugin_tests.py b/tests/plugin_tests.py
index cf874f1..5074cbf 100644
--- a/tests/plugin_tests.py
+++ b/tests/plugin_tests.py
@@ -221,7 +221,7 @@ class AddCopySpecLimitTests(unittest.TestCase):
 
     def test_single_file_under_limit(self):
         self.mp.add_copy_spec_limit("tests/tail_test.txt", 1)
-        self.assertEquals(self.mp.copy_specs, ['tests/tail_test.txt'])
+        self.assertEquals(self.mp.copy_paths, set(['tests/tail_test.txt']))
 
     def test_single_file_over_limit(self):
         fn = create_file(2) # create 2MB file, consider a context manager
-- 
1.9.3

