From 90c185e4c7f13a360576bf59768b03b41ad268e4 Mon Sep 17 00:00:00 2001
From: "Bryn M. Reeves" <bmr@redhat.com>
Date: Tue, 17 Jun 2014 17:18:41 +0100
Subject: [PATCH 72/72] [ovirt] add ovirt-scheduler-proxy logs

Signed-off-by: Bryn M. Reeves <bmr@redhat.com>
---
 sos/plugins/ovirt.py | 1 +
 1 file changed, 1 insertion(+)

diff --git a/sos/plugins/ovirt.py b/sos/plugins/ovirt.py
index cc8daaf..6cfb49c 100644
--- a/sos/plugins/ovirt.py
+++ b/sos/plugins/ovirt.py
@@ -84,6 +84,7 @@ class Ovirt(Plugin, RedHatPlugin):
             "/var/log/ovirt-engine",
             "/var/log/ovirt-engine-dwh",
             "/var/log/ovirt-engine-reports",
+            "/var/log/ovirt-scheduler-proxy",
             "/var/log/rhevm",
             "/etc/sysconfig/ovirt-engine",
             "/usr/share/ovirt-engine/conf",
-- 
1.9.3

