--- a/kexec-tools.spec
+++ b/kexec-tools.spec
@@ -1,6 +1,6 @@
 Name: kexec-tools
 Version: 2.0.4
-Release: 32%{?dist}
+Release: 32.1%{?dist}
 License: GPLv2
 Group: Applications/System
 Summary: The kexec/kdump userspace component.
@@ -374,6 +374,9 @@
 %endif
 
 %changelog
+* Thu Jun 26 2014 Martin Perina <mperina@redhat.com> - 2.0.4-32.1
+- support fence_kdump for generic cluster
+
 * Wed Apr 30 2014 Madper Xie <cxie@redhat.com> - 2.0.4-32
 - spec: fix wrong source id

