diff -u a/qemu-kvm.spec b/qemu-kvm.spec
--- a/qemu-kvm.spec	2015-05-08 08:40:19.675860954 +0200
+++ b/qemu-kvm.spec	2015-05-08 08:48:23.270332795 +0200
@@ -65,8 +65,9 @@
 # - conflicts_suffix:      used to prevent installation of both RHEL and RHEV
 
 %if %{rhev}
-    %global pkgsuffix %{rhev_suffix}
+    %global pkgsuffix -ev
     %global extra_provides_suffix %{nil}
+    %global rhev_provide_suffix %{rhev_suffix}
     %global conflicts_suffix %{rhel_suffix}
     %global obsoletes_version 15:0-0
 %else
@@ -79,6 +80,10 @@
 %define rhel_rhev_conflicts()                                         \
 Conflicts: %1%{conflicts_suffix}                                      \
 Provides: %1%{extra_provides_suffix} = %{epoch}:%{version}-%{release} \
+    %if 0%{?rhev_provide_suffix:1}                                     \
+Provides: %1%{rhev_provide_suffix} = %{epoch}:%{version}-%{release}   \
+Obsoletes: %1%{rhev_provide_suffix} < %{epoch}:%{version}-%{release}   \
+    %endif                                                            \
     %if 0%{?obsoletes_version:1}                                          \
 Obsoletes: %1 < %{obsoletes_version}                                      \
     %endif
@@ -1880,6 +1885,9 @@
 %endif
 
 %changelog
+* Fri May  8 2015 Sandro Bonazzola <sbonazzo@redhat.com> - 10:2.1.2-23.el7_1_1.2.1
+- Removing RH branding from package name
+
 * Tue Apr 21 2015 Miroslav Rezanina <mrezanin@redhat.com> - rhev-2.1.2-23.el7_1_1.2
 - kvm-block-Fix-max-nb_sectors-in-bdrv_make_zero.patch [bz#1203543]
 - Resolves: bz#1203543
