diff -u a/qemu-kvm.spec b/qemu-kvm.spec
--- a/qemu-kvm.spec	2018-10-24 08:55:10.534617004 +0200
+++ b/qemu-kvm.spec	2018-10-24 08:57:05.856546120 +0200
@@ -69,8 +69,9 @@
 # - conflicts_suffix:      used to prevent installation of both RHEL and RHEV
 
 %if %{rhev}
-    %global pkgsuffix %{rhev_suffix}
+    %global pkgsuffix -ev
     %global extra_provides_suffix %{nil}
+    %global rhev_provide_suffix %{rhev_suffix}
     %global conflicts_suffix %{rhel_suffix}
     %global obsoletes_version 15:0-0
     %global obsoletes_version2 15:0-0
@@ -101,12 +102,16 @@
 %endif                                                                 \
 %if 0%{?obsoletes_version2:1}                                          \
 Obsoletes: %1%{rhel_ma_suffix} < %{obsoletes_version2}                 \
+%endif                                                                 \
+%if 0%{?rhev_provide_suffix:1}                                         \
+Provides: %1%{rhev_provide_suffix} = %{epoch}:%{version}-%{release}    \
+Obsoletes: %1%{rhev_provide_suffix} < %{epoch}:%{version}-%{release}   \
 %endif
 
 Summary: QEMU is a machine emulator and virtualizer
 Name: %{pkgname}%{?pkgsuffix}
 Version: 2.10.0
-Release: 21%{?dist}.7
+Release: 21.el7_5.7.1
 # Epoch because we pushed a qemu-1.0 package. AIUI this can't ever be dropped
 Epoch: 10
 License: GPLv2+ and LGPLv2+ and BSD
@@ -2271,6 +2276,9 @@
 %endif
 
 %changelog
+* Tue Oct 23 2018 Sandro Bonazzola <sbonazzo@redhat.com> - ev-2.10.0-21.el7_5.7.1
+- Removing RH branding from package name
+
 * Tue Aug 28 2018 Miroslav Rezanina <mrezanin@redhat.com> - rhev-2.10.0-21.el7_5.7
 - kvm-aio-posix-Don-t-count-ctx-notifier-as-progress-when-.patch [bz#1619128]
 - kvm-aio-Do-aio_notify_accept-only-during-blocking-aio_po.patch [bz#1619128]
