diff -u a/qemu-kvm.spec b/qemu-kvm.spec
--- a/qemu-kvm.spec	2020-04-15 08:29:23.453751458 +0200
+++ b/qemu-kvm.spec	2020-04-15 08:31:54.328141673 +0200
@@ -71,8 +71,9 @@
 # - conflicts_suffix:      used to prevent installation of both RHEL and RHEV
 
 %if %{rhev}
-    %global pkgsuffix %{rhev_suffix}
+    %global pkgsuffix -ev
     %global extra_provides_suffix %{nil}
+    %global rhev_provide_suffix %{rhev_suffix}
     %global conflicts_suffix %{rhel_suffix}
     %global obsoletes_version 15:0-0
     %global obsoletes_version2 15:0-0
@@ -103,12 +104,16 @@
 %endif                                                                 \
 %if 0%{?obsoletes_version2:1}                                          \
 Obsoletes: %1%{rhel_ma_suffix} < %{obsoletes_version2}                 \
+%endif                                                                 \
+%if 0%{?rhev_provide_suffix:1}                                         \
+Provides: %1%{rhev_provide_suffix} = %{epoch}:%{version}-%{release}    \
+Obsoletes: %1%{rhev_provide_suffix} < %{epoch}:%{version}-%{release}   \
 %endif
 
 Summary: QEMU is a machine emulator and virtualizer
 Name: %{pkgname}%{?pkgsuffix}
 Version: 2.12.0
-Release: 44%{?dist}_8.1
+Release: 44.1%{?dist}_8.1
 # Epoch because we pushed a qemu-1.0 package. AIUI this can't ever be dropped
 Epoch: 10
 License: GPLv2 and GPLv2+ and CC-BY
@@ -1962,6 +1967,10 @@
 Patch899: kvm-util-add-slirp_fmt-helpers.patch
 # For bz#1798972 - CVE-2020-8608 qemu-kvm-rhev: QEMU: Slirp: potential OOB access due to unsafe snprintf() usages [rhel-7.8.z]
 Patch900: kvm-tcp_emu-fix-unsafe-snprintf-usages.patch
+# For bz#1822236 - Add support for newer glusterfs [rhel-7.8.z]
+Patch907: kvm-gluster-Handle-changed-glfs_ftruncate-signature.patch
+# For bz#1822236 - Add support for newer glusterfs [rhel-7.8.z]
+Patch908: kvm-gluster-the-glfs_io_cbk-callback-function-pointer-ad.patch
 
 BuildRequires: zlib-devel
 BuildRequires: glib2-devel
@@ -3080,6 +3089,8 @@
 %patch898 -p1
 %patch899 -p1
 %patch900 -p1
+%patch907 -p1
+%patch908 -p1
 
 # Fix executable permission for iotests
 chmod 755 $(ls tests/qemu-iotests/???)
@@ -3600,6 +3611,10 @@
 %endif
 
 %changelog
+* Tue Apr 14 2020 Sandro Bonazzola <sbonazzo@redhat.com> - ev-2.12.0-44.1.el7_8.1
+- Removing RH branding from package name
+- Add support for newer glusterfs in CentOS 7.8
+
 * Wed Mar 04 2020 Miroslav Rezanina <mrezanin@redhat.com> - 2.12.0-44.el7_8.1
 - kvm-util-add-slirp_fmt-helpers.patch [bz#1798972]
 - kvm-tcp_emu-fix-unsafe-snprintf-usages.patch [bz#1798972]
