--- a/qemu-kvm.spec	2016-08-19 14:17:25.955954703 +0200
+++ b/qemu-kvm.spec	2016-08-19 14:23:11.565360285 +0200
@@ -62,8 +62,9 @@
 # - extra_provides_suffix: used for dependency checking of other packages
 # - conflicts_suffix:      used to prevent installation of both RHEL and RHEV
 
-%global pkgsuffix %{rhev_suffix}
+%global pkgsuffix -ev
 %global extra_provides_suffix %{nil}
+%global rhev_provide_suffix %{rhev_suffix}
 %global conflicts_suffix %{rhel_suffix}
 %global obsoletes_version 15:0-0
 
@@ -71,12 +72,16 @@
 %define rhel_rhev_conflicts()                                         \
 Conflicts: %1%{conflicts_suffix}                                      \
 Provides: %1%{extra_provides_suffix} = %{epoch}:%{version}-%{release} \
+    %if 0%{?rhev_provide_suffix:1}                                    \
+Provides: %1%{rhev_provide_suffix} = %{epoch}:%{version}-%{release}   \
+Obsoletes: %1%{rhev_provide_suffix} < %{epoch}:%{version}-%{release}  \
+    %endif                                                            \
 Obsoletes: %1 < %{obsoletes_version}
 
 Summary: QEMU is a FAST! processor emulator
 Name:    %{pkgname}%{?pkgsuffix}
 Version: 2.3.0
-Release: 31%{?dist}.21
+Release: 31%{?dist}_2.21.1
 # Epoch because we pushed a qemu-1.0 package. AIUI this can't ever be dropped
 Epoch: 10
 License: GPLv2+ and LGPLv2+ and BSD
@@ -2560,6 +2565,9 @@
 %{_libdir}/pkgconfig/libcacard.pc
 
 %changelog
+* Fri Aug 19 2016 Sandro Bonazzola <sbonazzo@redhat.com> - ev-2.3.0-31.el7_2.21
+- Removing RH branding from package name
+
 * Tue Aug 02 2016 Miroslav Rezanina <mrezanin@redhat.com> - rhev-2.3.0-31.el7_2.21
 - kvm-block-iscsi-avoid-potential-overflow-of-acb-task-cdb.patch [bz#1358997]
 - Resolves: bz#1358997
