diff -u a/qemu-kvm.spec b/qemu-kvm.spec
--- a/qemu-kvm.spec	2019-01-29 14:06:35.695198972 +0100
+++ b/qemu-kvm.spec	2019-01-29 14:10:24.594224381 +0100
@@ -71,8 +71,9 @@
 # - conflicts_suffix:      used to prevent installation of both RHEL and RHEV
 
 %if %{rhev}
-    %global pkgsuffix %{rhev_suffix}
+    %global pkgsuffix -ev
     %global extra_provides_suffix %{nil}
+    %global rhev_provide_suffix %{rhev_suffix}
     %global conflicts_suffix %{rhel_suffix}
     %global obsoletes_version 15:0-0
     %global obsoletes_version2 15:0-0
@@ -103,12 +104,16 @@
 %endif                                                                 \
 %if 0%{?obsoletes_version2:1}                                          \
 Obsoletes: %1%{rhel_ma_suffix} < %{obsoletes_version2}                 \
+%endif                                                                 \
+%if 0%{?rhev_provide_suffix:1}                                         \
+Provides: %1%{rhev_provide_suffix} = %{epoch}:%{version}-%{release}    \
+Obsoletes: %1%{rhev_provide_suffix} < %{epoch}:%{version}-%{release}   \
 %endif
 
 Summary: QEMU is a machine emulator and virtualizer
 Name: %{pkgname}%{?pkgsuffix}
 Version: 2.12.0
-Release: 18%{?dist}_6.3
+Release: 18%{?dist}_6.3.1
 # Epoch because we pushed a qemu-1.0 package. AIUI this can't ever be dropped
 Epoch: 10
 License: GPLv2 and GPLv2+ and CC-BY
@@ -2425,6 +2430,9 @@
 %endif
 
 %changelog
+* Tue Jan 29 2019 Sandro Bonazzola <sbonazzo@redhat.com> - ev-2.12.0-18.el7_6.3.1
+- Removing RH branding from package name
+
 * Thu Dec 06 2018 Miroslav Rezanina <mrezanin@redhat.com> - 2.12.0-18.el7_6.3
 - kvm-include-Add-IEC-binary-prefixes-in-qemu-units.h.patch [bz#1640927]
 - kvm-hw-scsi-cleanups-before-VPD-BL-emulation.patch [bz#1640927]
