diff -u -r a/qemu-kvm.spec b/qemu-kvm.spec
--- a/qemu-kvm.spec	2018-05-02 16:30:42.699512459 +0200
+++ b/qemu-kvm.spec	2018-05-02 16:34:09.383719423 +0200
@@ -69,8 +69,9 @@
 # - conflicts_suffix:      used to prevent installation of both RHEL and RHEV
 
 %if %{rhev}
-    %global pkgsuffix %{rhev_suffix}
+    %global pkgsuffix -ev
     %global extra_provides_suffix %{nil}
+    %global rhev_provide_suffix %{rhev_suffix}
     %global conflicts_suffix %{rhel_suffix}
     %global obsoletes_version 15:0-0
     %global obsoletes_version2 15:0-0
@@ -101,12 +102,16 @@
 %endif                                                                 \
 %if 0%{?obsoletes_version2:1}                                          \
 Obsoletes: %1%{rhel_ma_suffix} < %{obsoletes_version2}                 \
+    %if 0%{?rhev_provide_suffix:1}                                     \
+Provides: %1%{rhev_provide_suffix} = %{epoch}:%{version}-%{release}    \
+Obsoletes: %1%{rhev_provide_suffix} < %{epoch}:%{version}-%{release}   \
+    %endif                                                             \
 %endif
 
 Summary: QEMU is a machine emulator and virtualizer
 Name: %{pkgname}%{?pkgsuffix}
 Version: 2.10.0
-Release: 21%{?dist}_5.1
+Release: 21.el7_5.1.2
 # Epoch because we pushed a qemu-1.0 package. AIUI this can't ever be dropped
 Epoch: 10
 License: GPLv2+ and LGPLv2+ and BSD
@@ -2181,6 +2186,12 @@
 %endif
 
 %changelog
+* Wed May  2 2018 Sandro Bonazzola <sbonazzo@redhat.com> - ev-2.10.0-21.el7_5.1.2
+- Fix Provides / Obsoletes for -rhev suffixes
+
+* Fri Apr 27 2018 Sandro Bonazzola <sbonazzo@redhat.com> - ev-2.10.0-21.el7_5.1.1
+- Removing RH branding from package name
+
 * Wed Mar 14 2018 Miroslav Rezanina <mrezanin@redhat.com> - 2.10.0-21.el7_5.1
 - kvm-hw-ppc-spapr_hcall-set-htab_shift-after-kvmppc_resiz.patch [bz#1554956]
 - kvm-memory-inline-some-performance-sensitive-accessors.patch [bz#1554929]
