diff -u a/qemu-kvm.spec b/qemu-kvm.spec
--- a/qemu-kvm.spec	2018-11-06 14:31:58.033056944 +0100
+++ b/qemu-kvm.spec	2018-11-06 14:36:58.926788887 +0100
@@ -71,8 +71,9 @@
 # - conflicts_suffix:      used to prevent installation of both RHEL and RHEV
 
 %if %{rhev}
-    %global pkgsuffix %{rhev_suffix}
+    %global pkgsuffix -ev
     %global extra_provides_suffix %{nil}
+    %global rhev_provide_suffix %{rhev_suffix}
     %global conflicts_suffix %{rhel_suffix}
     %global obsoletes_version 15:0-0
     %global obsoletes_version2 15:0-0
@@ -103,12 +104,16 @@
 %endif                                                                 \
 %if 0%{?obsoletes_version2:1}                                          \
 Obsoletes: %1%{rhel_ma_suffix} < %{obsoletes_version2}                 \
+%endif                                                                 \
+%if 0%{?rhev_provide_suffix:1}                                         \
+Provides: %1%{rhev_provide_suffix} = %{epoch}:%{version}-%{release}    \
+Obsoletes: %1%{rhev_provide_suffix} < %{epoch}:%{version}-%{release}   \
 %endif
 
 Summary: QEMU is a machine emulator and virtualizer
 Name: %{pkgname}%{?pkgsuffix}
 Version: 2.12.0
-Release: 18%{?dist}_6.1
+Release: 18%{?dist}_6.1.1
 # Epoch because we pushed a qemu-1.0 package. AIUI this can't ever be dropped
 Epoch: 10
 License: GPLv2 and GPLv2+ and CC-BY
@@ -2331,6 +2336,9 @@
 %endif
 
 %changelog
+* Tue Nov 06 2018 Sandro Bonazzola <sbonazzo@redhat.com> - ev-2.12.0-18.el7_6.1.1
+- Removing RH branding from package name
+
 * Thu Oct 11 2018 Miroslav Rezanina <mrezanin@redhat.com> - 2.12.0-18.el7_6.1
 - kvm-target-i386-cpu-Add-downstream-only-STIBP-CPUID-flag.patch [bz#1638077]
 - Resolves: bz#1638077
