diff -u a/qemu-kvm.spec b/qemu-kvm.spec
--- a/qemu-kvm.spec	2019-05-15 08:34:08.084089988 +0200
+++ b/qemu-kvm.spec	2019-05-15 08:38:05.044194048 +0200
@@ -71,8 +71,9 @@
 # - conflicts_suffix:      used to prevent installation of both RHEL and RHEV
 
 %if %{rhev}
-    %global pkgsuffix %{rhev_suffix}
+    %global pkgsuffix -ev
     %global extra_provides_suffix %{nil}
+    %global rhev_provide_suffix %{rhev_suffix}
     %global conflicts_suffix %{rhel_suffix}
     %global obsoletes_version 15:0-0
     %global obsoletes_version2 15:0-0
@@ -103,12 +104,16 @@
 %endif                                                                 \
 %if 0%{?obsoletes_version2:1}                                          \
 Obsoletes: %1%{rhel_ma_suffix} < %{obsoletes_version2}                 \
+%endif                                                                 \
+%if 0%{?rhev_provide_suffix:1}                                         \
+Provides: %1%{rhev_provide_suffix} = %{epoch}:%{version}-%{release}    \
+Obsoletes: %1%{rhev_provide_suffix} < %{epoch}:%{version}-%{release}   \
 %endif
 
 Summary: QEMU is a machine emulator and virtualizer
 Name: %{pkgname}%{?pkgsuffix}
 Version: 2.12.0
-Release: 18%{?dist}_6.5
+Release: 18%{?dist}_6.5.1
 # Epoch because we pushed a qemu-1.0 package. AIUI this can't ever be dropped
 Epoch: 10
 License: GPLv2 and GPLv2+ and CC-BY
@@ -2443,6 +2448,9 @@
 %endif
 
 %changelog
+* Wed May 15 2019 Sandro Bonazzola <sbonazzo@redhat.com> - ev-2.12.0-18.el7_6.5.1
+- Removing RH branding from package name
+
 * Wed Apr 10 2019 Danilo de Paula <ddepaula@redhat.com>- 2.12.0-18.el7_6.5
 - kvm-target-i386-define-md-clear-bit-rhev.patch [bz#1693219]
 - Resolves: bz#1693219
