--- a/qemu-kvm.spec	2021-04-08 13:47:49.078836900 +0200
+++ b/qemu-kvm.spec	2021-04-08 13:56:15.286572106 +0200
@@ -71,8 +71,9 @@
 # - conflicts_suffix:      used to prevent installation of both RHEL and RHEV
 
 %if %{rhev}
-    %global pkgsuffix %{rhev_suffix}
+    %global pkgsuffix -ev
     %global extra_provides_suffix %{nil}
+    %global rhev_provide_suffix %{rhev_suffix}
     %global conflicts_suffix %{rhel_suffix}
     %global obsoletes_version 15:0-0
     %global obsoletes_version2 15:0-0
@@ -103,12 +104,16 @@
 %endif                                                                 \
 %if 0%{?obsoletes_version2:1}                                          \
 Obsoletes: %1%{rhel_ma_suffix} < %{obsoletes_version2}                 \
+%endif                                                                 \
+%if 0%{?rhev_provide_suffix:1}                                         \
+Provides: %1%{rhev_provide_suffix} = %{epoch}:%{version}-%{release}    \
+Obsoletes: %1%{rhev_provide_suffix} < %{epoch}:%{version}-%{release}   \
 %endif
 
 Summary: QEMU is a machine emulator and virtualizer
 Name: %{pkgname}%{?pkgsuffix}
 Version: 2.12.0
-Release: 48%{?dist}_9.2
+Release: 48.1%{?dist}_9.2
 # Epoch because we pushed a qemu-1.0 package. AIUI this can't ever be dropped
 Epoch: 10
 License: GPLv2 and GPLv2+ and CC-BY
@@ -3657,6 +3662,9 @@
 %endif
 
 %changelog
+* Thu Apr 08 2021 Sandro Bonazzola <sbonazzo@redhat.com> - ev-2.12.0-48.1.el7_9.2
+- Removing RH branding from package name
+
 * Tue Dec 15 2020 Jon Maloy <jmaloy@redhat.com> - rhev-2.12.0-48.el7_9.2
 - kvm-Fix-use-afte-free-in-ip_reass-CVE-2020-1983.patch [bz#1837567]
 - kvm-hw-net-net_tx_pkt-fix-assertion-failure-in-net_tx_pk.patch [bz#1860962]
