--- a/qemu-kvm.spec	2015-09-25 13:59:02.267141816 +0200
+++ b/qemu-kvm.spec	2015-09-25 14:02:18.092809792 +0200
@@ -65,8 +65,9 @@
 # - conflicts_suffix:      used to prevent installation of both RHEL and RHEV
 
 %if %{rhev}
-    %global pkgsuffix %{rhev_suffix}
+    %global pkgsuffix -ev
     %global extra_provides_suffix %{nil}
+    %global rhev_provide_suffix %{rhev_suffix}
     %global conflicts_suffix %{rhel_suffix}
     %global obsoletes_version 15:0-0
 %else
@@ -79,6 +80,10 @@
 %define rhel_rhev_conflicts()                                         \
 Conflicts: %1%{conflicts_suffix}                                      \
 Provides: %1%{extra_provides_suffix} = %{epoch}:%{version}-%{release} \
+    %if 0%{?rhev_provide_suffix:1}                                     \
+Provides: %1%{rhev_provide_suffix} = %{epoch}:%{version}-%{release}   \
+Obsoletes: %1%{rhev_provide_suffix} < %{epoch}:%{version}-%{release}   \
+    %endif                                                            \
     %if 0%{?obsoletes_version:1}                                          \
 Obsoletes: %1 < %{obsoletes_version}                                      \
     %endif
@@ -86,7 +91,7 @@
 Summary: QEMU is a FAST! processor emulator
 Name: %{pkgname}%{?pkgsuffix}
 Version: 2.1.2
-Release: 23%{?dist}_1.9
+Release: 23%{?dist}_1.9.1
 # Epoch because we pushed a qemu-1.0 package. AIUI this can't ever be dropped
 Epoch: 10
 License: GPLv2+ and LGPLv2+ and BSD
@@ -1961,6 +1966,9 @@
 %endif
 
 %changelog
+* Fri Sep 25 2015 Sandro Bonazzola <sbonazzo@redhat.com> - 10:2.1.2-23.el7_1.9.1
+- Removing RH branding from package name
+
 * Fri Aug 28 2015 Miroslav Rezanina <mrezanin@redhat.com> - rhev-2.1.2-23.el7_1.9
 - kvm-vnc-fix-memory-corruption-CVE-2015-5225.patch [bz#1255897]
 - Resolves: bz#1255897
