diff -u a/qemu-kvm.spec b/qemu-kvm.spec
--- a/qemu-kvm.spec	2015-05-13 15:55:54.288445541 +0200
+++ b/qemu-kvm.spec	2015-05-13 15:54:08.403083081 +0200
@@ -65,8 +65,9 @@
 # - conflicts_suffix:      used to prevent installation of both RHEL and RHEV
 
 %if %{rhev}
-    %global pkgsuffix %{rhev_suffix}
+    %global pkgsuffix -ev
     %global extra_provides_suffix %{nil}
+    %global rhev_provide_suffix %{rhev_suffix}
     %global conflicts_suffix %{rhel_suffix}
     %global obsoletes_version 15:0-0
 %else
@@ -79,6 +80,10 @@
 %define rhel_rhev_conflicts()                                         \
 Conflicts: %1%{conflicts_suffix}                                      \
 Provides: %1%{extra_provides_suffix} = %{epoch}:%{version}-%{release} \
+    %if 0%{?rhev_provide_suffix:1}                                     \
+Provides: %1%{rhev_provide_suffix} = %{epoch}:%{version}-%{release}   \
+Obsoletes: %1%{rhev_provide_suffix} < %{epoch}:%{version}-%{release}   \
+    %endif                                                            \
     %if 0%{?obsoletes_version:1}                                          \
 Obsoletes: %1 < %{obsoletes_version}                                      \
     %endif
@@ -1883,6 +1888,9 @@
 %endif
 
 %changelog
+* Wed May 13 2015 Sandro Bonazzola <sbonazzo@redhat.com> - 10:2.1.2-23.el7_1_1.3.1
+- Removing RH branding from package name
+
 * Fri May 08 2015 Miroslav Rezanina <mrezanin@redhat.com> - rhev-2.1.2-23.el7_1.3
 - kvm-fdc-force-the-fifo-access-to-be-in-bounds-of-the-all.patch [bz#1219271]
 - Resolves: bz#1219271
