diff -u a/qemu-kvm.spec b/qemu-kvm.spec
--- a/qemu-kvm.spec	2017-10-18 17:00:27.153778471 +0200
+++ b/qemu-kvm.spec	2017-10-18 17:03:18.308914201 +0200
@@ -69,8 +69,9 @@
 # - conflicts_suffix:      used to prevent installation of both RHEL and RHEV
 
 %if %{rhev}
-    %global pkgsuffix %{rhev_suffix}
+    %global pkgsuffix -ev
     %global extra_provides_suffix %{nil}
+    %global rhev_provide_suffix %{rhev_suffix}
     %global conflicts_suffix %{rhel_suffix}
     %global obsoletes_version 15:0-0
 %else
@@ -84,6 +85,10 @@
 %define rhel_rhev_conflicts()                                         \
 Conflicts: %1%{conflicts_suffix}                                      \
 Provides: %1%{extra_provides_suffix} = %{epoch}:%{version}-%{release} \
+    %if 0%{?rhev_provide_suffix:1}                                    \
+Provides: %1%{rhev_provide_suffix} = %{epoch}:%{version}-%{release}   \
+Obsoletes: %1%{rhev_provide_suffix} < %{epoch}:%{version}-%{release}  \
+    %endif                                                            \
 %if 0%{?obsoletes_version:1}                                          \
 Obsoletes: %1 < %{obsoletes_version}                                  \
 %endif
@@ -91,7 +96,7 @@
 Summary: QEMU is a machine emulator and virtualizer
 Name: %{pkgname}%{?pkgsuffix}
 Version: 2.9.0
-Release: 16%{?dist}.8
+Release: 16.el7_4.8.1
 # Epoch because we pushed a qemu-1.0 package. AIUI this can't ever be dropped
 Epoch: 10
 License: GPLv2+ and LGPLv2+ and BSD
@@ -1378,6 +1383,9 @@
 %endif
 
 %changelog
+* Wed Oct 18 2017 Sandro Bonazzola <sbonazzo@redhat.com> - ev-2.9.0-16.el7_4.8.1
+- Removing RH branding from package name
+
 * Fri Sep 15 2017 Danilo Cesar Lemes de Paula <ddepaula@redhat.com> - rhev-2.9.0-16.el7_4.8
 - kvm-ui-move-qemu_input_linux_to_qcode.patch [bz#1482388]
 - kvm-ui-update-keymaps.patch [bz#1482388]
