diff -u a/qemu-kvm.spec b/qemu-kvm.spec
--- a/qemu-kvm.spec	2019-08-02 07:26:49.494354869 +0200
+++ b/qemu-kvm.spec	2019-08-02 07:30:43.891262412 +0200
@@ -71,8 +71,9 @@
 # - conflicts_suffix:      used to prevent installation of both RHEL and RHEV
 
 %if %{rhev}
-    %global pkgsuffix %{rhev_suffix}
+    %global pkgsuffix -ev
     %global extra_provides_suffix %{nil}
+    %global rhev_provide_suffix %{rhev_suffix}
     %global conflicts_suffix %{rhel_suffix}
     %global obsoletes_version 15:0-0
     %global obsoletes_version2 15:0-0
@@ -103,12 +104,16 @@
 %endif                                                                 \
 %if 0%{?obsoletes_version2:1}                                          \
 Obsoletes: %1%{rhel_ma_suffix} < %{obsoletes_version2}                 \
+%endif                                                                 \
+%if 0%{?rhev_provide_suffix:1}                                         \
+Provides: %1%{rhev_provide_suffix} = %{epoch}:%{version}-%{release}    \
+Obsoletes: %1%{rhev_provide_suffix} < %{epoch}:%{version}-%{release}   \
 %endif
 
 Summary: QEMU is a machine emulator and virtualizer
 Name: %{pkgname}%{?pkgsuffix}
 Version: 2.12.0
-Release: 18%{?dist}_6.7
+Release: 18%{?dist}_6.7.1
 # Epoch because we pushed a qemu-1.0 package. AIUI this can't ever be dropped
 Epoch: 10
 License: GPLv2 and GPLv2+ and CC-BY
@@ -2461,6 +2466,9 @@
 %endif
 
 %changelog
+* Fri Aug 02 2019 Sandro Bonazzola <sbonazzo@redhat.com> - ev-2.12.0-18.el7_6.7.1
+- Removing RH branding from package name
+
 * Tue May 28 2019 Miroslav Rezanina <mrezanin@redhat.com> - 2.12.0-18.el7_6.7
 - kvm-slirp-check-sscanf-result-when-emulating-ident.patch [bz#1669070]
 - kvm-slirp-fix-big-little-endian-conversion-in-ident-prot.patch [bz#1669070]
