diff -u a/qemu-kvm.spec b/qemu-kvm.spec
--- a/qemu-kvm.spec	2015-09-08 10:10:09.417779110 +0200
+++ b/qemu-kvm.spec	2015-09-08 10:12:30.309296208 +0200
@@ -65,8 +65,9 @@
 # - conflicts_suffix:      used to prevent installation of both RHEL and RHEV
 
 %if %{rhev}
-    %global pkgsuffix %{rhev_suffix}
+    %global pkgsuffix -ev
     %global extra_provides_suffix %{nil}
+    %global rhev_provide_suffix %{rhev_suffix}
     %global conflicts_suffix %{rhel_suffix}
     %global obsoletes_version 15:0-0
 %else
@@ -79,6 +80,10 @@
 %define rhel_rhev_conflicts()                                         \
 Conflicts: %1%{conflicts_suffix}                                      \
 Provides: %1%{extra_provides_suffix} = %{epoch}:%{version}-%{release} \
+    %if 0%{?rhev_provide_suffix:1}                                     \
+Provides: %1%{rhev_provide_suffix} = %{epoch}:%{version}-%{release}   \
+Obsoletes: %1%{rhev_provide_suffix} < %{epoch}:%{version}-%{release}   \
+    %endif                                                            \
     %if 0%{?obsoletes_version:1}                                          \
 Obsoletes: %1 < %{obsoletes_version}                                      \
     %endif
@@ -86,7 +91,7 @@
 Summary: QEMU is a FAST! processor emulator
 Name: %{pkgname}%{?pkgsuffix}
 Version: 2.1.2
-Release: 23%{?dist}_1.8
+Release: 23%{?dist}_1.8.1
 # Epoch because we pushed a qemu-1.0 package. AIUI this can't ever be dropped
 Epoch: 10
 License: GPLv2+ and LGPLv2+ and BSD
@@ -1958,6 +1963,9 @@
 %endif
 
 %changelog
+* Tue Sep  8 2015 Sandro Bonazzola <sbonazzo@redhat.com> - 10:2.1.2-23.el7_1.8.1
+- Removing RH branding from package name
+
 * Thu Aug 06 2015 Miroslav Rezanina <mrezanin@redhat.com> - rhev-2.1.2-23.el7_1.8
 - kvm-qemu_ram_foreach_block-pass-up-error-value-and-down-.patch [bz#1248382]
 - kvm-rdma-Fix-qemu-crash-when-IPv6-address-is-used-for-mi.patch [bz#1248382]
