--- a/qemu-kvm.spec	2015-09-25 14:02:54.113932666 +0200
+++ b/qemu-kvm.spec	2015-09-25 14:03:04.962969669 +0200
@@ -1085,7 +1085,7 @@
 Group:          Development/Libraries
 Requires:       libcacard%{?pkgsuffix} = %{epoch}:%{version}-%{release}
 # older qemu-img has vscclient which is now in libcacard-tools
-Requires:       qemu-img >= 3:1.3.0-5
+Requires:       qemu-img%{?pkgsuffix} >= 3:1.3.0-5
 
 %rhel_rhev_conflicts libcacard-tools
 
