diff -u a/qemu-kvm.spec b/qemu-kvm.spec
--- a/qemu-kvm.spec	2017-04-20 15:25:00.689393160 +0200
+++ b/qemu-kvm.spec	2017-04-20 15:25:26.355092031 +0200
@@ -1417,7 +1417,7 @@
 Group:          Development/Libraries
 Requires:       libcacard%{?pkgsuffix} = %{epoch}:%{version}-%{release}
 # older qemu-img has vscclient which is now in libcacard-tools
-Requires:       qemu-img >= 3:1.3.0-5
+Requires:       qemu-img%{?pkgsuffix} >= 3:1.3.0-5
 
 %rhel_rhev_conflicts libcacard-tools
 
