diff -u a/qemu-kvm.spec b/qemu-kvm.spec
--- a/qemu-kvm.spec	2015-09-08 10:13:18.285472289 +0200
+++ b/qemu-kvm.spec	2015-09-08 10:13:29.674514088 +0200
@@ -1083,7 +1083,7 @@
 Group:          Development/Libraries
 Requires:       libcacard%{?pkgsuffix} = %{epoch}:%{version}-%{release}
 # older qemu-img has vscclient which is now in libcacard-tools
-Requires:       qemu-img >= 3:1.3.0-5
+Requires:       qemu-img%{?pkgsuffix} >= 3:1.3.0-5
 
 %rhel_rhev_conflicts libcacard-tools
 
Solo in b: qemu-kvm.spec.orig
