diff -u a/qemu-kvm.spec b/qemu-kvm.spec
--- a/qemu-kvm.spec	2017-01-20 14:59:31.911056361 +0100
+++ b/qemu-kvm.spec	2017-01-20 14:59:48.398850648 +0100
@@ -1358,7 +1358,7 @@
 Group:          Development/Libraries
 Requires:       libcacard%{?pkgsuffix} = %{epoch}:%{version}-%{release}
 # older qemu-img has vscclient which is now in libcacard-tools
-Requires:       qemu-img >= 3:1.3.0-5
+Requires:       qemu-img%{?pkgsuffix} >= 3:1.3.0-5
 
 %rhel_rhev_conflicts libcacard-tools
 
Solo in b: qemu-kvm.spec.orig
