diff -u a/qemu-kvm.spec b/qemu-kvm.spec
--- a/qemu-kvm.spec	2016-08-19 14:25:24.788271405 +0200
+++ b/qemu-kvm.spec	2016-08-19 14:26:30.038248312 +0200
@@ -1483,7 +1483,7 @@
 Group:          Development/Libraries
 Requires:       libcacard%{?pkgsuffix} = %{epoch}:%{version}-%{release}
 # older qemu-img has vscclient which is now in libcacard-tools
-Requires:       qemu-img >= 3:1.3.0-5
+Requires:       qemu-img%{?pkgsuffix} >= 3:1.3.0-5
 
 %rhel_rhev_conflicts libcacard-tools
 
